components/encryption/src/io.rs

```diff
@@ -741,13 +734,13 @@ mod tests {
             EncryptionMethod::Sm4Ctr,
         ];
         let mut plaintext = vec![0; 10240];
-        OsRng.fill_bytes(&mut plaintext);
+        rand::rand_bytes(&mut plaintext).unwrap();
         let offsets = [1024, 1024 + 1, 10240 - 1];
         let sizes = [1024, 8000];
         let written = vec![0; 10240];
         for method in methods {
-            let key = generate_data_key(method);
-            let iv = Iv::new_ctr();
+            let (_, key) = generate_data_key(method).unwrap();
```

Comment:
The error can be ignored ? 
Can we think the related `rand` function in `openssl::rand` always generate a bunch of random bytes as expected ?

Reply:
It should not be ignored, as an error meaning the OpenSSL CSPRNG can not generate secure random numbers.
The error is rare on recent platform.